Templates/CSharp/Desktop/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="mstestProjectTemplate" Version="15.0.26130.00" Language="en-US" Publisher="Microsoft" />
+    <Identity Id="mstestProjectTemplate" Version="15.0.27010.00" Language="en-US" Publisher="Microsoft" />
```

Comment:
just curious, where did we get this number from?

Reply:
Picked up the latest build number of VS

Reply:
I see, should it be 15.5 then?